services/frontend-service/src/ui/Batch.tsx

```diff
@@ -116,6 +175,20 @@ export const ConfirmationDialogProvider = (props: ConfirmationDialogProviderProp
             />
         )) ?? null;
 
+    const conflictMessage = conflict ? (
+        <TextField
```

Comment:
I chose it for the warning and the outline properties. Simply listing text could get really messy. esp. with the locks being there too.

Reply:
Material UI has some other elements that create a structure. I guess an alert message would be the right thing in this case https://mui.com/components/alert/

Reply:
i'm not sure why, but i can't get it in the red color. (I set the prop to error but the component didn't change.)
(my current theory is because some different library version. because I encountered other variations between what I can actually do and what's on the documentation)

Reply:
@hannesg